config/routes.yml

```diff
@@ -40,3 +40,10 @@ admin_link_block_delete:
   defaults:
     _controller: 'PrestaShop\Module\LinkList\Controller\Admin\Improve\Design\LinkWidgetController::deleteAction'
     _legacy_controller: AdminLinkWidget
+
+admin_link_block_update_positions:
+  path: /link_widget/update_positions
```

Comment:
any reason to use underscore instead of hyphen?

Reply:
absolutely none ^^

Reply:
It's always better to use hyphen instead of underscore in urls ;) (even if this page isn't in seo context, let's keep good practices :D)